packages/wrangler/src/api/dev.ts

```diff
@@ -176,7 +177,7 @@ export async function unstable_dev(
 		bundle: options?.bundle,
 		compatibilityDate: options?.compatibilityDate,
 		compatibilityFlags: options?.compatibilityFlags,
-		ip: options?.ip,
+		ip: options?.ip ?? "127.0.0.1",
```

Comment:
we're using `"win32" ? "127.0.0.1" : "localhost"` as the default port in all our code. Is this something we want to consider here as well?